client/rpc/finality/src/lib.rs

```diff
@@ -13,11 +13,15 @@
 
 // You should have received a copy of the GNU General Public License
 // along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+pub use fc_db::sql::{self};
```

Comment:
Is line is probably no needed

Reply:
Weird that there's no "unused import" warning. Anyway you are right ->

```suggestion
```